rules/windows/process_creation/proc_creation_win_office_exec_from_trusted_locations.yml

```diff
@@ -0,0 +1,51 @@
+title: Potentially Suspicious Office Document Executed From Trusted Location
+id: f99abdf0-6283-4e71-bd2b-b5c048a94743
+status: experimental
+description: Detects the execution of an office application that points to a document that is located in a trusted location. Attackers often used this to avoid macro security and execute their malicious code.
```

Comment:
```suggestion
description: Detects the execution of an Office application that points to a document that is located in a trusted location. Attackers often used this to avoid macro security and execute their malicious code.
```